schema/schema_template.py

```diff
@@ -338,4 +338,38 @@ def validate_comments(self, key, value):
         check_string_length(value, DESCRIPTION_STRING_LENGTH, key)
         return value
 
+class Measurement(Base):
+    # Template table. This is a placeholder for a table that would store measurements such as parallax.
+
+    __tablename__ = 'Measurement'
+    source = Column(
+        String(100),
+        ForeignKey("Sources.source", ondelete="cascade", onupdate="cascade"),
+        nullable=False,
+        primary_key=True,
+    )
+
+    # units should be replaced with the unit of measurement.
+    # e.g., for parallax measured in milliarcseconds, this would read parallax_mas
+    measurement_units = Column(Float, nullable=False)
+
+    # units should be replaced with the unit of measurement.
+    # e.g., for parallax measured in milliarcseconds, this would read parallax_error_mas
+    measurement_error_units = Column(Float) # todo: make asymmetric errors
```

Comment:
Do we actually need the units in the error column if we already have it in the actual value?
Regardless: we can group both the value and error columns together with the same type of comment message.

Reply:
As for asymmetric errors, I think a comment guiding users is enough (eg, appending _upper or _lower). 

This opens up a complicated matter of users wanting to sometimes store symmetric, sometimes asymmetric, and may wish to convert back and forth. I think that's beyond the scope of this PR and is something we need to clarify how much we want to support in general.

Reply:
True! No longer have units.